### DISCUSSION TITLE
I can't delete old Personal access token from server before I changed ip address

### DISCUSSION THREAD

**Discussion by JR2012thman** [victorbalssa/abacus#281] [Q&A] [answered]
Title says most, in the manage credentials for the IOS app I can't seem to delete the old credential and the application remembers even if I delete it, is this a bug? I really hope I am just not understanding how it wants me to delete it but I am at a loss.

**Comment by victorbalssa** [comment, marked as answer]
@JR2012thman It's undocumented but you will need to `Go to credential` screen (in the Settings menu) and then click on Edit top right corner to delete a credential.

**Reply by JR2012thman** [reply]
The go to credential screen takes me back to the Home Screen. Is that normal?

**Reply by JR2012thman** [reply]
Turns out you need to have two credentials stored to get the edit button. Thanks
